src/utils.ts

```diff
@@ -608,7 +612,16 @@ export async function constructAndLoadIoTProject(
     telemetryContext: TelemetryContext, isTriggeredWhenExtensionLoad = false) {
   const scaffoldType = ScaffoldType.Workspace;
 
-  const projectFileRootPath = getFirstWorkspaceFolderPath();
+  let projectFileRootPath = '';
+  try {
+    projectFileRootPath = getFirstWorkspaceFolderPath();
+  } catch (error) {
+    if (isTriggeredWhenExtensionLoad) {
```

Comment:
Why the behavior is different when isTriggeredWhenExtensionLoad is true? what is that scenario?

Reply:
When a vscode window is open, before user trigger any workbench command, we will try load the current folder if any. In this situation, it is the `isTriggeredWhenExtensionLoad == true` scenario. In this situation, we do not want to throw error when try to load project since user may not want to use workbench at all.